clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/model/KubernetesServerGroup.groovy

```diff
@@ -102,7 +108,51 @@ class KubernetesServerGroup implements ServerGroup, Serializable {
     this.namespace = namespace
   }
 
-  KubernetesServerGroup(ReplicaSet replicaSet, String account, List<Event> events, HorizontalPodAutoscaler autoscaler) {
+
+  KubernetesServerGroup(V1beta1DaemonSet daemonSet, String account, List<Event> events) {
```

Comment:
I was kind-of hoping to avoid doing this translation, and relying on the raw k8s resources instead. This would come at the cost of updating the k8s provider with a non-backward compatible change, or releasing a v2 of the k8s provider.